validator/src/commands/exit/mod.rs

```diff
@@ -0,0 +1,94 @@
+use {
+    crate::{admin_rpc_service, cli::DefaultArgs, commands},
+    clap::{value_t_or_exit, App, Arg, ArgMatches, SubCommand},
+    solana_clap_utils::input_validators::{is_parsable, is_valid_percentage},
+    std::{path::Path, process::exit},
+};
+
+pub fn command(default_args: &DefaultArgs) -> App<'_, '_> {
+    SubCommand::with_name("exit")
+        .about("Send an exit request to the validator")
+        .arg(
+            Arg::with_name("force")
+                .short("f")
+                .long("force")
+                .takes_value(false)
+                .help(
+                    "Request the validator exit immediately instead of waiting for a restart window",
```

Comment:
This one technically exceeds 100 characters now. If the linter doesn't yell at you, I'm fine with it